enhancements/sig-policy/62-namespace-labelselector/README.md

```diff
@@ -0,0 +1,252 @@
+# Namespace `LabelSelector` for `ConfigurationPolicies`
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+Implement `MatchLabels` and `MatchExpressions` (collectively known as `LabelSelector`) for
+`ConfigurationPolicies` in order to further filter the list of namespaces to which an object should
+be applied.
+
+## Motivation
+
+Currently the `ConfigurationPolicy` has a `spec.namespaceSelector` that includes `include` and
```

Comment:
Note that this shouldn't really be limited to `ConfigurationPolicy` since this feature is also needed for the certificate policies too.  If we move to a common library for our controllers it may fix this anyway, but wanted to make sure we aim for some consistency here since this is really needed for cert policies too.